backport rtw88_usb from wireless-next

981213 opened the pull request:
tested with COMFAST CF-924AC v2(RTL8812BU) on Linux 5.4 (Ubuntu 20.04)

lwfinger commented:
Thanks. Merged.

lwfinger commented:
Note: I got merge problems when I pulled the updated version as I had already added that code to my base here. The problem was in the Makefile. Because this repo does not use Kconfig, the symbols like CONFIG_RTW88_8822BU do not exist, thus builds use 
obj-m      += rtw88_8822bu.o rather than
obj-$(CONFIG_RTW88_8822BU)      += rtw88_8822bu.o.

In other words, we build them as modules unconditionally.

I also added the new modules to the installation, and signing process.

981213 commented:
> Note: I got merge problems when I pulled the updated version as I had already added that code to my base here. The problem was in the Makefile. Because this repo does not use Kconfig, the symbols like CONFIG_RTW88_8822BU do not exist, thus builds use obj-m += rtw88_8822bu.o rather than obj-$(CONFIG_RTW88_8822BU) += rtw88_8822bu.o.
> 
> In other words, we build them as modules unconditionally.

I did manually changed these when applying the upstream patches.
 
> I also added the new modules to the uninstall, and signing process.

But I didn't notice this one. Thanks for fixing it :)
